app/src/main/res/values-it/strings.xml

```diff
@@ -54,19 +54,25 @@
     <string name="comment_edit_type_your_comment">Digita il tuo commento</string>
     <string name="comment_mention_node_block">Blocca %1$s</string>
     <string name="comment_mention_node_copy_permalink">Copia permalink</string>
+    <string name="comment_mention_node_copy_comment">Copia testo del commento</string>
     <string name="comment_mention_node_go_to">Vai a</string>
     <string name="comment_mention_node_permalink_copied">Permalink copiato</string>
+    <string name="comment_mention_node_comment_copied">Testo copiato</string>
     <string name="comment_mention_node_report_comment">Segnala commento</string>
     <string name="comment_mention_node_view_source">Mostra markdown</string>
+    <string name="comment_next_parent">Vai al prossimo commento padre</string>
```

Comment:
I wonder if we could find a better word for `parent`.
`superiore`could be an option, but in some context it may be mistaken for "the one above".
Although, thinking of it, in file system navigation this stuff gets usually translated that way (e.g. vai alla cartella superiore).
In this case it could become:
```suggestion
    <string name="comment_next_parent">Vai al commento superiore successivo</string>
```

Reply:
I agree about the potential confusion indeed. There's no good translation I can think of, so maybe let's leave your original translation and call it a day.
I had also thought of `commento radice` for a sec, but technically speaking it would be a bit different, plus as you imply it's also not a very common concept outside of data structures (although most of lemmy users ATM are probably geeks running Linux anyway, so...)

Reply:
Yeah I can't think of a better translation either. I liked your `successivo` better than `prossimo` so I updated it, but I guess let's leave `padre` in there until someone can come up with a better idea.

Also, I think this text is not visible in the UI. I'm guessing it's for blind / low vision users. Not to throw them under the bus, but probably almost nobody will read or hear it.